examples/ui/button.rs

```diff
@@ -6,6 +6,7 @@
 #![allow(clippy::type_complexity)]
 
 use bevy::{prelude::*, winit::WinitSettings};
+use bevy_internal::render::camera::Viewport;
```

Comment:
I don't think you are allowed to import from `bevy_internal` in examples. I think you can just import from `bevy`. If not, you may need to make this type public, add it to the prelude or something.